content/docs/ai/_index.md

```diff
@@ -0,0 +1,88 @@
+
+### Multi-provider support
+
+AI Gateway supports multiple LLM providers, including OpenAI, Anthropic, and Gemini. For the full list, see the [Supported LLM providers](#supported-llm-providers) section. 
+
+### Model traffic management
+
+With AI Gateway, you can manage traffic to different models across LLM providers. Traffic management includes features such as the following:
+
+* **A/B model testing**: Perform controlled experiments between two or more model versions. This feature allows users to evaluate model quality and performance with selective cohorting and routing to determine which model performs better for specific use cases or prompts.
+* **Canary model rollout**: Deploy a new model version to a subset of traffic while continuing to route the majority of traffic to an existing model. This rollout pattern helps ensure stability and correctness of a new model before a full rollout.
+* **Model failover**: Failover is a way to keep services running smoothly by automatically switching to a backup system when the main one fails or becomes unavailable. With AI Gateway, you can set up failover for the models of the LLM providers that you want to prioritize. If the main model from one provider goes down, slows, or has any issue, the system quickly switches to a backup model from that same provider. This keeps the service running without interruptions.
+
+### Prompt enrichment
+
+Prompts are basic building blocks for guiding LLMs to produce relevant and accurate responses. By effectively managing both system prompts, which set initial guidelines, and user prompts, which provide specific context, you can significantly enhance the quality and coherence of the model's outputs. AI Gateway allows you to pre-configure and refactor system and user prompts, extract common AI provider settings so that you can reuse them across requests, dynamically append or prepend prompts to where you need them, and overwrite default settings on a per-route level.
+
+### Prompt guards
+
+Prompt guards are mechanisms that ensure that prompt-based interactions with a language model are secure, appropriate, and aligned with the intended use. These mechanisms help to filter, block, monitor, and control LLM inputs and outputs to filter offensive content, prevent misuse, and ensure ethical and responsible AI usage. With AI Gateway, you can set up prompt guards to block unwanted requests to the LLM provider and mask sensitive data.
+
+### Advanced enterprise features {#enterprise}
+
+For the following features, consider an enterprise solution such as [Gloo AI Gateway](https://docs.solo.io/gateway/latest/ai/).
+
+* Centralized credential management
+* Access control
+* Token-weighted rate limiting
+* Retrieval augmented generation (RAG)
+* Semantic caching
+* Function calling
```

Comment:
we will need to link this to a hypothetical /enterprise in order to comply with CNCF neutrality rules.

Reply:
Yes, good point. We discussed in scrum, and I will just remove it.